test/test_baseline_agents.py

```diff
@@ -25,28 +26,39 @@
     not baseline_installed, reason="baseline sub-module not installed"
 )
 def test_ppo_agents():
+
     agent_config = ppo_agents.get_default_config()
     agent_config.MODEL_PATH = ""
+    agent_config.defrost()
     config_env = habitat.get_config(config_paths=CFG_TEST)
     if not os.path.exists(config_env.SIMULATOR.SCENE):
         pytest.skip("Please download Habitat test data to data folder.")
 
     benchmark = habitat.Benchmark(config_paths=CFG_TEST)
 
     for input_type in ["blind", "rgb", "depth", "rgbd"]:
-        config_env.defrost()
-        config_env.SIMULATOR.AGENT_0.SENSORS = []
-        if input_type in ["rgb", "rgbd"]:
-            config_env.SIMULATOR.AGENT_0.SENSORS += ["RGB_SENSOR"]
-        if input_type in ["depth", "rgbd"]:
-            config_env.SIMULATOR.AGENT_0.SENSORS += ["DEPTH_SENSOR"]
-        config_env.freeze()
-        del benchmark._env
-        benchmark._env = habitat.Env(config=config_env)
-        agent_config.INPUT_TYPE = input_type
-
-        agent = ppo_agents.PPOAgent(agent_config)
-        habitat.logger.info(benchmark.evaluate(agent, num_episodes=10))
+        for resolution in [256, 384]:
```

Comment:
Use `resolution` for `@pytest.mark.parametrize` like [here](https://github.com/facebookresearch/habitat-api/blob/master/test/test_baseline_trainers.py#L29).

Reply:
That would require reconstructing the agent and benchmark for every iteration? I think there is a reason this is already done in a loop before I added the code (it would make the test a lot longer).

Reply:
Makes sense, do you want to test when `h <> w`?